packages/validator/templates/configSecurity.json

```diff
@@ -0,0 +1,98 @@
+{
+  "config": {
+    "ois": {
+      "__arrayItem": {
+        "__oisSpecs": {}
+      }
+    },
+    "triggers": {
+      "__optional": {
+        "request": {
+          "__arrayItem": {
+            "endpointId": {
+              "__regexp": "^0x[a-fA-F0-9]{64}$"
+            },
+            "oisTitle": {},
```

Comment:
Is `oisTitle` supposed to map to some object in the `ois` array? If so, I think there should be a `__require` statement like this, no?

Hmm ... although I'm not sure the `__require` statement currently has this functionality: to check if there exists some condition in *any* element in an array.

Speaking of: @bbenligiray why is `ois` in `config.json` an array instead of an object with `oisTitle` as a key?

Reply:
Because, if `triggers.oisTitle` does in fact map to some `ois[].title` (pls confirm), then that title should be easily index-able.

Reply:
Because `oisTitle` is already a field in the OIS object and using the title both as the key and a field of the object that is indexed by that key is weird. Same argument also applies for `triggers` for example, we should use `endpointId`s as keys but we just keep a list of objects and loop through them to find what we're looking for (but that's not any more difficult to implement and results in a neater format).